### DISCUSSION TITLE
Transform feature request: add "absolute" as a Binary transform math operator

### DISCUSSION THREAD

**Discussion by ethirolle** [grafana/grafana#38134] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Add "absolute" as a Binary transform math operator (i.e., like a difference operator, but always positive).

**Why is this needed**:
Below is direct quote from customer request:

> The ability to perform differentials is available now, but I am getting positive and negative values when I just need the absolute value of the spread between my numbers.

**Comment by ethirolle** [comment]
ZenDesk reference: ticket # 15220

**Comment by HogeBlekker** [comment]
Would be useful for my use case as well.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
